smacrobs_web/src/main/webapp/WEB-INF/Views/dashboard.jsp

```diff
@@ -332,138 +139,52 @@
 			</script>
 
 		<!-- #section:basics/sidebar -->
-		<div id="sidebar" class="sidebar                  responsive">
+		<div id="sidebar" class="sidebar responsive">
 			<script type="text/javascript">
 					try{ace.settings.check('sidebar' , 'fixed')}catch(e){}
 				</script>
 
-			<div class="sidebar-shortcuts" id="sidebar-shortcuts">
-				<div class="sidebar-shortcuts-large" id="sidebar-shortcuts-large">
-					<button class="btn btn-success">
-						<i class="ace-icon fa fa-signal"></i>
-					</button>
-
-					<button class="btn btn-info">
-						<i class="ace-icon fa fa-pencil"></i>
-					</button>
-
-					<!-- #section:basics/sidebar.layout.shortcuts -->
-					<button class="btn btn-warning">
-						<i class="ace-icon fa fa-users"></i>
-					</button>
-
-					<button class="btn btn-danger">
-						<i class="ace-icon fa fa-cogs"></i>
-					</button>
-
-					<!-- /section:basics/sidebar.layout.shortcuts -->
-				</div>
-
-				<div class="sidebar-shortcuts-mini" id="sidebar-shortcuts-mini">
-					<span class="btn btn-success"></span> <span class="btn btn-info"></span>
-
-					<span class="btn btn-warning"></span> <span class="btn btn-danger"></span>
-				</div>
-			</div>
+		
 			<!-- /.sidebar-shortcuts -->
 
 			<!--SSR put the profile details here-->
 			<div id="profile">
 
 
+				<ul class="nav nav-list">
+					<li class=""><a href="index.html"> <i class="menu-icon fa fa-user"></i> 
+					<span class="menu-text"> ${fullName} </span>
+					</a> <b class="arrow"></b></li>
 
+					<li class=""><a href="index.html"> <i class="menu-icon fa fa-intersex"></i> 
+							<span class="menu-text">Gender: ${gender} </span>
+					</a> <b class="arrow"></b></li>
 
+					<li class=""><a href="index.html"> <i class="menu-icon fa fa-calendar"></i> 
+							<span class="menu-text">Age: ${age} </span>
```

Comment:
Make sure you get appropriate default values from server. 'null' is lot better than '0'